packages/mobile/package.json

```diff
@@ -62,6 +65,7 @@
     "array.prototype.flat": "1.2.5",
     "audius-client": "1.3.10",
     "babel-plugin-transform-remove-console": "6.9.4",
+    "@babel/runtime": "7.15.4",
```

Comment:
had to move this to dependencies to get the app working w/ Hermes

Reply:
so intriguing